python/dglke/data/dataset/KGDataset.py

```diff
@@ -117,7 +117,7 @@ def read_relation(self, relation_path):
 
         return relation2id, len(relation2id)
 
-    def read_triple(self, path, mode, skip_first_line=False, format=[0,1,2]):
+    def read_triple(self, path, mode, skip_first_line=False, format=[0,1,2], inverse_rel=False):
```

Comment:
Why we need inverse_rel?

Reply:
Something trick for Hyperbolic Embedding. It treats head->rel->tail and tail->rel->head differently and auto-augment the data to have tail->inv_rel->head.